src/com/owncloud/android/ui/asynctasks/CopyAndUploadContentUrisTask.java

```diff
@@ -238,7 +238,7 @@ protected void onPostExecute(ResultCode result) {
             listener.onTmpFilesCopied(result);
 
         } else {
-            Log_OC.i(TAG, "User left Uploader activity before the temporal copies were finished ");
+            Log_OC.i(TAG, "User left ReceiveExternalFilesActivity activity before the temporal copies were finished ");
```

Comment:
This log should not include the name of the left activity now that could be any; just something like "the caller activity" or "the parent activity"